tracer/test/test-applications/integrations/Samples.Kafka/Properties/launchSettings.json

```diff
@@ -12,7 +12,33 @@
         "CORECLR_PROFILER": "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}",
         "CORECLR_PROFILER_PATH": "$(SolutionDir)shared/bin/monitoring-home/osx/Datadog.Trace.ClrProfiler.Native.dylib",
         "COR_PROFILER": "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}",
-        "DD_TRACE_DEBUG": "1"
+        "DD_TRACE_DEBUG": "1",
+        "DD_TRACE_AGENT_PORT": "8136"
```

Comment:
Maybe you want to remove these two or nah?

Reply:
Good call yeah 👍